packages/syft/src/syft/service/user/user_service.py

```diff
@@ -210,7 +210,12 @@ def get_current_user(
                 SyftError(message="User not found!")
         return SyftError(message=str(result.err()))
 
-    @service_method(path="user.update", name="update", roles=GUEST_ROLE_LEVEL)
+    @service_method(
+        path="user.update",
+        name="update",
+        roles=GUEST_ROLE_LEVEL,
+        autosplat=["user_update"],
```

Comment:
@koenvanderveen I reverted this for now. This breaks `user_test.py` even after updating the syntax. `UserUpdate` has a couple of validators which don't get applied if using `autosplat`.

https://github.com/OpenMined/PySyft/blob/687aefd64076f312e26ebd855221baf200d418f2/packages/syft/src/syft/service/user/user.py#L114-L122

Reply:
👍